examples/04_vector_field/01_3d_vector_field.py

```diff
@@ -0,0 +1,70 @@
+"""
+Generating a Random 3D Vector Field
+-----------------------------------
+
+In this example we are going to generate a random 3D vector field with a
+Gaussian covariance model. The mesh on which we generate the field will be
+externally defined and it will be generated by PyVista.
+"""
+import numpy as np
+import gstools as gs
+import pyvista as pv
+
+pv.set_plot_theme("document")
+
+###############################################################################
+# create a uniform grid with PyVista
+nx, ny, nz = 40, 30, 10
+mesh = pv.UniformGrid((nx, ny, nz), (1., 1., 1.), (-10., 0., 0.))
```

Comment:
would prefere:
```python
dim, spacing, origin = (40, 30, 10), (1, 1, 1), (-10, 0, 0)
mesh = pv.UniformGrid(dim, spacing, origin)
```
Following the documentation of pyvista.

Reply:
Yeah, that makes things clearer.